plugins/PHP/Plugins/Common/PluginsDefines.php

```diff
@@ -38,14 +38,23 @@
 define("PP_PHP_METHOD","1501");
 define("PP_PHP","1500");
 define("PP_MYSQL","2101");
+define("PP_MSSQL_SERVER", "2200");
+define("PP_ORACLE", "2300");
+define("PP_CUBRID", "2400");
+define("PP_POSTGRESQL", "2500");
+define("PP_CASSANDRA", "2600");
+define("PP_MONGODB", "2651");
 define("PP_KAFKA","8660");
 define("PP_REDIS","8200");
 define("PP_REDIS_REDISSON","8203");
 define("PP_REDIS_REDISSON_INTERNAL","8204");
+define("PP_RABBITMQ", "8300");
+define("PP_ACTIVEMQ", "8310");
+define("PP_ARCUS", "8100");
 define("PP_MEMCACHED","8050");
 define("PP_PHP_REMOTE",9700);
 
-define("PP_HEADER_NGINX_PROXY","Pinpoint-ProxyNginx");
+define("PP_HEADER_NGINX_PROXY","HTTP_PINPOINT_PROXYNGINX");
```

Comment:
Pinpoint-ProxyNginx can't change to HTTP_PINPOINT_PROXYNGINX. 
Nginx will keep the case, while apache not.

Reply:
Ok, I've revert the change.